config/webhook/kustomization.yaml

```diff
@@ -4,6 +4,7 @@ kind: Kustomization
 resources:
 - service.yaml
 - manifests.yaml
+- storage_quota_webhook_configuration.yaml
```

Comment:
Is there any reason for not generating this in the `manifests.yaml` file using a kubebuilder tag as other existing webhook configs?

Reply:
we don't want to generate with a kubebuilder tag, because this is for the storage quota webhook, not for our webhook. we want the storage quota webhook to be called for create/update of vm, not any of our webhooks. the storage quota webhook will then call our "webhook" to get the capacity for storage policy usage. so, it's not included in `manifests.yaml` because we don't want it generated, and if it's hard-coded there, then it'll get removed upon generation

Reply:
Got it. In that case it makes sense to keep it separate file to have its CA annotation and other fields correctly.